doc/api/repl.md

```diff
@@ -375,6 +375,16 @@ The `replServer.displayPrompt` method is primarily intended to be called from
 within the action function for commands registered using the
 `replServer.defineCommand()` method.
 
+### replServer.clearBufferedCommand()
+<!-- YAML
+added: v8.1.0
```

Comment:
Should this probably be REPLACEME till landing? Anyway, even 8.1.1 has not this fix.